features/object_mapper/using/README.md

```diff
@@ -0,0 +1,267 @@
+
+Extra columns in the executed query will be ignored. About fields in the mapped object that are not in the `ResultSet`'s rows, setters won't be called to set the field so their value will be the value after invocation of the default's class constructor.
+
+Here is an example with previously defined tables and mapped entities : 
+
+```java
+ResultSet results = session.execute("SELECT * FROM user");
+Result<User> users = mapper.map(results);
+for (User u : users){
+    System.out.println("User : " + u.getUserId());
+}
+```
+
+`Result` class is similar to `ResultSet` but for a given mapped class. It provides methods `one()`, `all()`, `iterator()`, `getExecutionInfo()` and `isExhausted()`. Be careful, the `ResultSet` given in parameter will be stored in the `Result` class. This means that consumed rows in `ResultSet` will be considered consumed in the `Result` also.
+
+Because of previous warnings (`ResultSet` consumption, and empty fields), we recommend trying to prevent the use this `map()` method, and either stick to [CRUD operations](#basic-crud-operations), or use [queries generated by the mapper](#generating-queries-from-mapper), or use [Accessors](#accessors).
+
+***
+
+#### UDT Mapper
+
+A UDT will be automatically mapped when it is part of a table-level class (as demonstrated by `User#address` in our example). However, there might be cases where it is useful to get back a UDTValue from a non-mapper query and map it to its domain class. It is possible to get a UDTMapper from the MappingManager:
+
+```java
+UDTMapper<Address> addressMapper = manager.getUDTMapper(Address.class);
+Address address = addressMapper.map(udtValue);
+```
+
+### Accessors
+
+The object mapper provides ways to easily execute customized queries and bind them into mapped objects, these are called `Accessor`s. Accessors define functions written by the client to perform an operation in the database, with potential parameters, and maps results into objects. `@Query` contains the query to be executed in Cassandra and is followed by a method declaration.
+
+Here is an example with the previously defined `User` class :
+
+```java
+@Accessor
+public interface UserAccessor {
+    @Query("SELECT * FROM user")
+    Result<User> getAll();
+}
+```
+
+The return type is important here, the results of the query will be automatically either mapped in a single object or in a collection of objects.
+
+The `MappingManager` can then process this class and generate an implementation :
+
+```java
+UserAccessor userAccessor = manager.createAccessor(UserAccessor.class);
+User user = userAccessor.getOne(uuid);
+```
+
+Like mappers, accessors are cached at the manager level and thus, are thread-safe/sharable.
+
+--------------
+
+Named or unnamed bind markers can be used in a `@Query`, when using a bind marker in a `@Query`, the corresponding accessor's method can accept parameters. Parameters are specified with `@Param` annotation before each parameter of the method : 
```

Comment:
`@Param` is only necessary for named parameters.
When using positional parameters, the order of the method's argument must match the order of the bind markers in the query.